pom.xml

```diff
@@ -12,7 +12,7 @@
 
     <groupId>io.github.jhipster</groupId>
     <artifactId>jhipster-dependencies</artifactId>
-    <version>0.0.5</version>
+    <version>0.0.6</version>
```

Comment:
shouldn't it be changed only when there is a release ?

Reply:
I think that it should be done right after a release, as master correspond to the next iteration. Ideally, I should have tagged it 0.0.6-SNAPSHOT., then remove the snapshot flag at the release (or configure the Maven release plugin to automate it)

Reply:
Hehe, Julien just merged this and now the released version went from 0.0.5 to 0.0.7 :-)

I agree it would be better to let the maven-release plugin do this, but for now the process seems to be that it gets done by hand just before a release, and therefore PRs should not change the version props.

Reply:
Oh yes this has always been done by hand, did you do anything to change that? I want to control this, it's too important, at least for the moment

Reply:
The PR you just merged from Tcharl changed the property from 0.0.5 to 0.0.6, and then you bumped it to 0.0.7 for release. So now there's a gap in the history.

Reply:
Yes I know, it's an error and now it's too late. Why did this PR bump the version number? It should not, I'm the one doing the releases. We only bump the version number at the very end, it should be the latest commit before a release. This also prevents errors on my side (which I already did!!), as this means I can't do a release by mistake.